Build/Dependencies/react-elm-components/example/Chat.elm

```diff
@@ -0,0 +1,139 @@
+port module Chat exposing (main)
```

Comment:
Why commit these examples?

Reply:
Like I said, it's more a joke than serious and I simply cloned the `react-elm-components`-package to change one line in their code.